### PR TITLE
fix space

### PR DESCRIPTION
**Pull request by RevuraV**

### CODE REVIEW

File: client/.stylelintrc.yml [hunk 1]
```diff
@@ -9,13 +9,13 @@ rules:
   declaration-no-important: true
   font-family-no-duplicate-names: true
   function-name-case: 'lower'
-  max-nesting-depth: 3
+  max-nesting-depth: 4
```
Comment (anna-kuzii) [inline client/.stylelintrc.yml]: nope

File: client/.stylelintrc.yml [hunk 2]
```diff
@@ -9,13 +9,13 @@ rules:
...
   no-duplicate-selectors: true
   no-empty-first-line: true
   order/properties-alphabetical-order: null
   selector-max-id: 0
   selector-no-qualifying-type: null
-  selector-max-compound-selectors: 3
+  selector-max-compound-selectors: 4
```
Comment (anna-kuzii) [inline client/.stylelintrc.yml]: nope

File: client/src/components/Post/style.scss [hunk 3]
```diff
@@ -32,6 +32,17 @@
     &::-webkit-scrollbar-thumb {
       background-color: $color-dove-gray;
     }
+
+    @media(min-width: $break-md) {
```
Comment (anna-kuzii) [inline client/src/components/Post/style.scss]: not good

File: client/src/components/Post/style.scss [hunk 4]
```diff
@@ -32,6 +32,17 @@
...
+      .modal > & {
```
Comment (anna-kuzii) [inline client/src/components/Post/style.scss]: WTF???

**Review by anna-kuzii — CHANGES_REQUESTED** [review]

File: client/.stylelintrc.yml (hunk 1 above)
Comment (RevuraV) [reply client/.stylelintrc.yml]: fixed

File: client/.stylelintrc.yml (hunk 2 above)
Comment (RevuraV) [reply client/.stylelintrc.yml]: fixed

File: client/src/components/Post/style.scss (hunk 4 above)
Comment (Volodia1999) [reply client/src/components/Post/style.scss]: How to write it better?

**Review by anna-kuzii — APPROVED** [review]

**Review by IhorKalyta — APPROVED** [review]

**Pull request merged**